components/Reference/Filters.tsx

```diff
@@ -45,6 +47,13 @@ const Filters = ({ onSetFilter, isPrecompiled = false }: Props) => {
     setSearchFilter(option)
   }
 
+  const handleAltK = (event: KeyboardEvent) => {
+    if (event.altKey && (event.key === 'k' || event.key === 'K')) {
```

Comment:
This should work too and is more concise.

```suggestion
    if (event.altKey && (event.key.toLowerCase() === 'k')) {
```